common/web/gesture-recognizer/README.md

```diff
@@ -0,0 +1,22 @@
+# Keyman:  Gesture Recognizer Module
+The Original Code is (C) 2022-2022 SIL International
+
+The Gesture Recognizer module is an internal component for all of Keyman's Web-based OSKs.  At this time, this is only referenced by
+KeymanWeb, seen within this repo at /web/.
+
+## Minimum Compilation Requirements
+
+* A local installation of [Node.js](https://nodejs.org/) v8.9+.
+	* Builds will call `npm install` to automatically install further necessary build dependencies.
+
+	* Linux users can run the following to update to LTS version of nodejs
+	
+```
+sudo apt-get install python-software-properties
+curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash -
+sudo apt-get install nodejs
+```
+
+**********************************************************************
+
+The build script may be found at src/build.sh.
```

Comment:
According to the [build script documentation](https://docs.google.com/document/d/1xoQaO7ZcICNm2q_C35Cm7Z9TgYq8d2XErSJblVPXM28/edit#bookmark=id.qffvhw7t9lqb), build.sh should not be in the `src/` folder but in component root.

Note: this was not all that clear in the build script docs so I've just clarified it there. And yes, this is currently inconsistent at present (_some_ web components under `src/` or `source/`, everything else at product/component root), and I will be aiming to fix this for existing components when I do build script consolidation.

Reply:
I was referencing https://docs.google.com/document/d/1gWlgDA_re33i_ny4FiBrjuZLJLBJqlIAbspRxI55FQk/edit#heading=h.gvz7zega9tlw, which seems to contradict that documentation.